torch/nn/_functions/thnn/upsampling.py

```diff
@@ -137,10 +137,6 @@ def backward(self, grad_output):
         )
         return grad_input
 
-    def __setstate__(self, state):
```

Comment:
FYI, I removed this because:

1) we don't generally pickle autograd functions. (Non-leaf variables can't be saved)
2) `_tuple()` isn't defined so this code never worked